### PR TITLE
Mutlisig graph tree

### PR DESCRIPTION
**Pull request by bassemmagdy**
- Adding multisig Tree to the account page

### CODE REVIEW

File: src/store/Account.ts [hunk 1]
```diff
@@ -389,6 +396,20 @@ export default {
                 )
                 .toPromise();
 
+            // sorted array to be represented in multisig tree
+            const getMultisigAccountGraphArraySortedPromise = repositoryFactory
```
Comment (yilmazbahadir) [inline src/store/Account.ts:400]: Why are we making the same API call here again? Is it just to get the result as sorted? If so why not we use the previous result and sort it?

Comment (bassemmagdy) [reply src/store/Account.ts:400]: makes sense

File: src/store/Account.ts [hunk 2]
```diff
@@ -425,6 +433,7 @@ export default {
             commit('multisigAccountsInfo', multisigAccountsInfo);
             commit('currentAccountMultisigInfo', currentAccountMultisigInfo);
             commit('currentSignerMultisigInfo', currentSignerMultisigInfo);
+            // commit('multisigAccountGraph', multisigAccountGraph);
```
Comment (yilmazbahadir) [inline src/store/Account.ts:436]: Can we remove this line?

File: src/components/AccountMultisigGraph/AccountMultisigGraphTs.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    visible: boolean;
+    public multisigAccountGraphInfo: [];
+    public knownAccounts: AccountModel[];
+
+    get multisigGraphTree(): any[] {
+        if (this.multisigAccountGraphInfo) {
+            const tree = [];
+            this.multisigAccountGraphInfo.map((level: []) => {
+                level.map((entry) => {
+                    // @ts-ignore
```
Comment (yilmazbahadir) [inline src/components/AccountMultisigGraph/AccountMultisigGraphTs.ts:48]: We might get rid of the ts-ignores by specifying the correct type for the multisigAccountGraphInfo variable? multisigAccountGraphInfo: MultisigAccountInfo[][] maybe?

File: src/components/AccountMultisigGraph/AccountMultisigGraphTs.ts [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                    const selected = this.account.address === entry.accountAddress.address ? true : false;
```
Comment (yilmazbahadir) [inline src/components/AccountMultisigGraph/AccountMultisigGraphTs.ts:49]: Do we need a ternary operator here? instead => `const selected = this.account.address === entry.accountAddress.address;`

**Review by rg911 — APPROVED** [review]

**Review by yilmazbahadir — APPROVED** [review]

**Pull request merged**